Sources/Extensions/Double/Double+Rounding.swift

```diff
@@ -23,36 +23,37 @@
 import Foundation
 
 public extension Double {
-
+    
     /**
      Type of rounding double value
-
+     
```

Comment:
please undo these useless changes
this might help: XCode -> Preferences -> Text Editing -> While Editing -> Check both

Reply:
hmm. i've checked and tried to reverse changes